docs/UserGuide.adoc

```diff
@@ -48,7 +48,16 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 ====
 *Command Format*
 
-* `<>` denotes compulsory field. Example: `add <attraction>`
+* Words in UPPER_CASE are the parameters to be supplied by the user
```

Comment:
Are upper case really used? The use is not consistent. For instance, you can find multiple "index" without and with full caps.